src/lib/updateGrantsRecipients.js

```diff
@@ -168,11 +235,22 @@ export async function processFiles(hashSumHex) {
         endDate: valueFromXML(program.grant_program_end_date),
         status: valueFromXML(program.program_status),
         name: valueFromXML(program.program_name),
-      }));
+        // Get grant personnel.
+        programPersonnel: await getProgramPersonnel(
+          parseInt(grantAgencyMap[program.grant_agency_id], 10),
+          parseInt(program.grant_program_id, 10),
+          program,
+        ),
+      })));
+
+      // Extract an array of all grant personnel to update.
+      const programPersonnel = programsForDb.map((p) => p.programPersonnel).flat();
```

Comment:
not suggesting a change, just an FYI that there's a `flatMap` operator that does this :upside_down_face:

e.g. `programsForDb.flatMap(({ programPersonnel }) => programPersonnel);`

Reply:
Thought I tried that with no luck. Let me try again.

Reply:
Yea it worked odd wonder what I did wrong the first time.